DeAceroDLabs/marketplace-frontend
# [DMP-743] Update Search body request

## What is done

[[DMP-743] Search Request no Funciona](https://deacero.atlassian.net/browse/DMP-743)

## How is done

1. Updated the Search body request, from `orderBy` to `orderByParam` which is how Roobyn expects it

## How to test

1. Run the project via `npm start`.
2. Search for something.
3. Products should load.
